docs/getting-started/delete-on-dockstore.rst

```diff
@@ -0,0 +1,44 @@
+Deleting an entry on Dockstore
+===================================
+
+Tutorial Goals
+--------------
+
+-  Deleting an entry
+-  Uninstalling the Dockstore GitHub App from the source GitHub repo
+
+This tutorial walks through the process of deleting entries from Dockstore.
+This action is permanent and after you delete the workflow, it will no longer appear on Dockstore.
+
+Requirements for Deletion
+-------------------------
+
+An entry can only be deleted if it satisfies all the following:
+
+-  Has never been published,
+-  Was created on/after April 1, 2022, and
+-  Is not a checked or checker workflow.
+
+
+Deleting an Entry
+------------------
+
+Navigate to ``/my-tools``, ``/my-workflows``, ``/my-notebooks``, or ``/my-services`` by going to My Dashboard then selecting the desired option in the left sidebar.
+In the sidebar accordion, find the github repository that your entry is in and click the entry you would like to delete.
```

Comment:
github -> GitHub

Reply:
I still see `github`. Maybe you just haven't pushed yet, but flagging it just in case.